### PR TITLE
Updated url branch for 2.0.x

### PR DESCRIPTION
**Pull request by rigazilla**
Updated readme for branch 2.0.x

### CODE REVIEW

**Comment by dmvolod** [comment]
lgtm, thanks
@rigazilla , we also pay attention on update operator image tag inside deploy.yaml and regen install bundle

File: deploy/operator.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ spec:
       serviceAccountName: infinispan-operator
       containers:
         - name: infinispan-operator
-          image: jboss/infinispan-operator:latest
+          image: quay.io/infinispan/operator:2.0.x
```
Comment (dmvolod) [inline deploy/operator.yaml:18]: @rigazilla we should also regen install-bundle.yaml

Comment (rigazilla) [reply deploy/operator.yaml:18]: yep, done thanks!

**Comment by rigazilla** [comment]
merged, thanks

**Pull request merged**